HEC.FDA.Importer/AsciiImportExport/AsciiImport.cs

```diff
@@ -855,6 +856,14 @@ void ProcessOcctypeFields()
 
             }
         }
+
+        private void FixRatioParameters()
```

Comment:
can i get a comment to describe what this function is doing? It's unclear to me.

Reply:
sure thing!